.nais/nais-q0.yml

```diff
@@ -39,6 +39,9 @@ spec:
         kvPath: /serviceuser/data/dev/srvsoknadsmottaker
       - mountPath: /secrets/innsending-data
         kvPath: secret/team-soknad/innsending/basicauth
+      - mountPath: /var/run/secrets/nais.io/resend
+        kvPath: /kv/preprod/fss/soknadsmottaker/resend
```

Comment:
Hvis q0 og q1 deler samme path, kan vi "risikere" at det kjøres re-sending i et miljø der vi ikke ønsker det. Kan derfor være lurt å skille på q0 vs q1 ved å legge eventuelle resendingsdata miljøspesifikke mapper slik som gjort for q1 i nais-q1.yml.

Reply:
Höres meget rimelig ut. Siden det ikke finnes noen sådan struktur i [Vault](https://vault.adeo.no/ui/vault/secrets/kv%2Fpreprod%2Ffss/show/soknadsmottaker/resend) så tenkte jeg at det var bedre at fjerne referense til miljö.

Men du tenker at vi bör ha `kvPath: /kv/preprod/fss/soknadsmottaker/resend/q0` respektive `kvPath: /kv/preprod/fss/soknadsmottaker/resend/q1` i config-filene våres?

Reply:
ja